patch/llpcPatchPeepholeOpt.cpp

```diff
@@ -582,6 +582,12 @@ void PatchPeepholeOpt::visitPHINode(
 
             pResult = InsertElementInst::Create(pResult, pNewPhiNode, pElementIndex, "", pInsertPos);
 
+            // Make sure the same incoming blocks have identical incoming values.
+            // If we have already inserted an incoming arc for a basic block,
+            // reuse the same value in the future incoming arcs from the same block.
+            SmallDenseMap<BasicBlock *, Value *, 8> incomingBlockToValue;
```

Comment:
SmallDenseMap<BasicBlock *, Value *, 8> -> SmallDenseMap<BasicBlock*, Value*, 8>
BasicBlock * -> BasicBlock*
Value * -> Value*
We treat "Type*" as a whole entity.

incomingBlockToValue -> incomingPairMap